akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala

```diff
@@ -135,7 +134,16 @@ final class ClusterSettings(val config: Config, val systemName: String) {
     cc.getMillisDuration("quarantine-removed-node-after")
       .requiring(_ > Duration.Zero, "quarantine-removed-node-after must be > 0")
 
-  val AllowWeaklyUpMembers: Boolean = cc.getBoolean("allow-weakly-up-members")
+  val WeaklyUpAfter: FiniteDuration = {
+    val key = "allow-weakly-up-members"
+    toRootLowerCase(cc.getString(key)) match {
+      case "off" => Duration.Zero
+      case "on"  => 5.seconds // for backwards compatibility when it wasn't a duration
```

Comment:
Shouldn't this be `3.seconds`?

Maybe we should even do `gossip-interval * 3` to really keep the previous behavior.

Reply:
True if we want to be completely compatible, but I would like to sneak in a change of the default since the previous 3 seconds was too tight.